pkg/polaris/aws/aws.go

```diff
@@ -353,6 +382,27 @@ func (a API) AddAccount(ctx context.Context, account AccountFunc, features []cor
 	return akkount.ID, nil
 }
 
+func (a API) addAccountOutpost(ctx context.Context, config account, options options) error {
+	a.log.Print(log.Trace)
+
+	accountInit, err := aws.Wrap(a.client).ValidateAndCreateCloudAccount(ctx, options.outpostAccountID, options.outpostAccountID, []core.Feature{core.FeatureOutpost})
+	if err != nil {
+		return fmt.Errorf("failed to validate account: %s", err)
+	}
+
+	err = aws.Wrap(a.client).FinalizeCloudAccountProtection(ctx, config.cloud, options.outpostAccountID, options.outpostAccountID, []core.Feature{core.FeatureOutpost}, options.regions, accountInit)
+	if err != nil {
+		return fmt.Errorf("failed to add account: %s", err)
+	}
+
+	err = awsUpdateStack(ctx, a.client.Log(), *config.config, accountInit.StackName, accountInit.TemplateURL)
+	if err != nil {
+		return fmt.Errorf("failed to update CloudFormation stack: %s", err)
+	}
+
+	return nil
+}
+
```

Comment:
I think we can drop this function and just use the `addAccountWithCFT` function.